File: TimeTableServer/DAL/Models/SubjectForCycle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DAL.Models
{
    public class SubjectForCycle
    {
        public int Id { get; set; }
        public int IdCycle { get; set; }
        public int idSubject { get; set; }
        public int NumberOfLessonsPerWeek { get; set; }
        public int idTeacher { get; set; }

    }
}

File: TimeTableClient/src/app/Classes/SubjectForCycle.ts
export class SubjectForCycle{
constructor( 
    public Id: number,
    public IdCycle: number,
    public idSubject: number,
    public NumberOfLessonsPerWeek: number,
    public idTeacher: number){}
}

export class IscedualValidate{
    constructor(
        public  teacherName: number,
        public  numHourDif: number
    ){}
}


